src/main/java/com/taakestan/video/VideoTransformer.java

```diff
@@ -0,0 +1,95 @@
+package com.taakestan.video;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.apache.commons.exec.CommandLine;
+import org.apache.commons.exec.DefaultExecutor;
+import org.apache.commons.exec.ExecuteWatchdog;
+import org.apache.commons.exec.OS;
+import org.apache.commons.exec.PumpStreamHandler;
+
+/**
+ * @author Omid Pourhadi : omidpourhadi [AT] gmail [DOT] com
+ *
+ */
+public class VideoTransformer
+{
+
+    private static final Logger log = Logger.getLogger(VideoTransformer.class.getName());
+
+    
+    /**
+     * @param fielPath : full file path in filesystem
+     * @param outputDirectory : 
+     * @return : null or outputdirectory + fileName_web.mp4 file
+     */
+    public static File transform(String fielPath, String outputDirectory)
```

Comment:
`inputVideoFile` would be a better name for the first argument and it's also more general if you expect type as File rather than String.